tools/bpf2c/bpf2c.cpp

```diff
@@ -116,6 +116,12 @@ get_program_info_type_hash(const std::vector<int32_t>& actual_helper_ids, const
                 hash_t::append_byte_range(
                     byte_range, program_info->program_type_specific_helper_prototype[index].arguments[argument]);
             }
+            // This check for flags is temporary, until https://github.com/microsoft/ebpf-for-windows/issues/3429 is
+            // fixed.
+            if (program_info->program_type_specific_helper_prototype[index].flags.reallocate_packet != 0) {
+                hash_t::append_byte_range(
+                    byte_range, program_info->program_type_specific_helper_prototype[index].flags);
```

Comment:
is it possible to round each bit field to 8-bit boundary, and add the corresponding bytes to the hash? So for `reallocate_packet`, we can add 1 byte of data to the hash.